packages/ember/index.js

```diff
@@ -16,6 +16,21 @@ module.exports = {
     },
   },
 
+  getAddonConfig(app) {
+    const config = require(app.options.configPath)(app.env);
```

Comment:
Is it always given that `configPath` will be at least defined? If not, then we should either wrap it in `try/catch` or `if (!app.options || app.options.configPath)`

Reply:
config path should be always defined, since it's pulling the conventional `config/environment.js` which should always exist, but I think we can wrap this in try/catch anyway in case someone has an edge case 👍